village/src/components/SmurfForm.js

```diff
@@ -16,15 +16,25 @@ class SmurfForm extends Component {
   }
 
```

Comment:
### When to add state into a component
  1.  If a component has multiple child components that rely on data coming from the same place, that component may need state.  
       - `App.js` has child components that need its data, so it is stateful
  2.  If a component "owns" its own data, or if information is being created inside of it, this is a good use case for a stateful component.  
       -  Because `SmurfForm` owns those `input` boxes, which rely on state, it's okay to use state here